en/smart_batteries/rotoye_batmon.md

```diff
@@ -0,0 +1,23 @@
+# Rotoye Batmon
+
+Rotoye Batmon is a smartification kit for off-the-shelf Lithium-Ion and LiPo batteries. It can be purchased as a standalone unit or factory-assembled into a smart-battery. 
+
+<img src="../../assets/hardware/smart_batteries/rotoye_batmon/smart-battery-rotoye-pack.jpeg" alt="Pack" width="360"/>
+<br/><br/>
+<img src="../../assets/hardware/smart_batteries/rotoye_batmon/smart-battery-rotoye.jpg" alt="Board" width="360"/>
+
+
+## Purchase
+
+[Rotoye Store](https://rotoye.com/batmon/): Batmon kits, custom smart-batteries, and accessories
+
+
+## Wiring/Connections
```

Comment:
1. I think it is worth here copying in the PX4 setup and https://github.com/rotoye/batmon_reader#px4
2. Also ideally the additional images
   ![image](https://user-images.githubusercontent.com/5368500/107160915-43c3c080-69ed-11eb-91d9-a0b579874b2f.png)
   Or even better an end to end showing battery, batmon, pixhawk 4 ...
3. If you don't want to add the info about the cell to batmon wiring, add a note saying - "Information on how to connect batteries to the battery monitor is covered here.

Reply:
I went ahead and made those changes; I used the above image rather than the large one of the XT90.

Reply:
As for the cell connection, new versions of Batmon now connect directly to the standard 2.54mm balance leads on LiPo/LiIo batteries; that diagram on Github is out of date

Reply:
If you can show an updated version of the battery connection that would be really cool, though not in any way mandatory.